### ISSUE TITLE
How to extract the feature vector from net [darknet.cfg]?

### ISSUE THREAD

**Issue by yanchao12122** [pjreddie/darknet#225]
how can I convert this layer to a vector 1x1024 to represent the feature of an image?
#################################
[convolutional]    # how can I convert this layer to a vector 1x1024 to represent the feature of an image?
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

[convolutional]
filters=1000
size=1
stride=1
pad=1
activation=leaky

[avgpool]

[softmax]
groups=1

[cost]
type=sse

**Comment by sayanmutd** [comment]
How did you managed to extract features? @yanchao12122

**Comment by zpmmehrdad** [comment]
@yanchao12122 Hi,

How did you do that?
